setup.py

```diff
@@ -12,9 +12,10 @@
     history = history_file.read()
 
 requirements = [
-    'morphys==1.0',
-    'pymultihash==0.8.2',
-    'pyrsistent==0.13.0',
+    'base58',
```

Comment:
Please pin these dependencies to their specific versions. It is important as we don't want our code to break when any of our dependencies update their package with backward incompatible change or bugs.

Reply:
Okay, but at the same time maybe let's hook up a dependency updater so that we can immediately know whether new dependencies would break the project.